CITATIONS.md

```diff
@@ -62,6 +62,10 @@
 
 - [FILTLONG](https://github.com/rrwick/Filtlong)
 
+- [falco](https://doi.org/10.12688/f1000research.21142.2)
+
+> de Sena Brandine G and Smith AD. Falco: high-speed FastQC emulation for quality control of sequencing data. F1000Research 2021, 8:1874
```

Comment:
We will need to do a pass before release to make sure these are all consistent but fine for now